### PR TITLE
[EDA] People & Groups: Account Naming Frontend

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes

# Issues Closed

# New Metadata
### Apex Classes
AccountModelSettingsController
AccountModelSettingsController_TEST
AccountNamingSettingsVModel
AccountNamingSettingsVModel_TEST
AccountNamingSettingsVMapper
AccountNamingSettingsVMapper_TEST

### Custom Labels
stgAdminAccountNamingTitle
stgAdminAccountCustomName
stgAdminAccountCustomNameHelp
stgHHAccountNamingTitle
stgHHAccountCustomName
stgHHAccountCustomNameHelp

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/accountModelSettings/accountModelSettings.js [hunk 1]
```diff
@@ -117,10 +183,94 @@ export default class AccountModelSettings extends LightningElement {
...
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Admin_Account_Naming_Format__c",
+            settingsValue: event.detail.value,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleAdministrativeAccountCustomNamingChange(event) {
```
Comment (m-kinkade) [inline force-app/main/default/lwc/accountModelSettings/accountModelSettings.js:201]: @MatthewBlanski is it acceptable to have this method fire every time something is typed in the custom name format field?

File: force-app/main/default/lwc/accountModelSettings/accountModelSettings.js [hunk 2]
```diff
@@ -117,10 +183,94 @@ export default class AccountModelSettings extends LightningElement {
...
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Household_Account_Naming_Format__c",
+            settingsValue: event.detail.value,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleHouseholdAccountCustomNamingChange(event) {
```
Comment (m-kinkade) [inline force-app/main/default/lwc/accountModelSettings/accountModelSettings.js:236]: @MatthewBlanski is it acceptable to have this method fire every time something is typed in the custom name format field?

File: force-app/main/default/lwc/accountModelSettings/accountModelSettings.js (hunk 1 above)
Comment (m-kinkade) [reply force-app/main/default/lwc/accountModelSettings/accountModelSettings.js:201]: added timeout as discussed

File: force-app/main/default/lwc/accountModelSettings/accountModelSettings.js (hunk 2 above)
Comment (m-kinkade) [reply force-app/main/default/lwc/accountModelSettings/accountModelSettings.js:234]: added timeout as discussed

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
A few changes. The most important is that the combobox is not expanding to as far as we need it to. The others are mentioned below. Some console logs need pruning and we can genericize even further.

File: force-app/main/default/lwc/accountModelSettings/accountModelSettings.js [hunk 3]
```diff
@@ -77,8 +78,14 @@ export default class AccountModelSettings extends LightningElement {
         hhAutomaticAccountNamingTitle: automaticHHNaming,
         hhAutomaticAccountNamingDescription: automaticHHNamingHelpText,
         accountNamingComboboxCustomOption: acctNamingOther,
+        tellMeMoreLink: stgTellMeMoreLink,
     };
 
+    accNamingHyperLink =
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/accountModelSettings/accountModelSettings.js:84]: Clever. This certainly works!

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
Passes QA - Merging

**Pull request merged**